2018-04-28 13:56:33,101 ERROR: Exception on /favicon.ico [GET] [in /Users/andreasantarlasci/Apps/microblog/venv/lib/python3.6/site-packages/flask/app.py:1560]
Traceback (most recent call last):
  File "/Users/andreasantarlasci/Apps/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/andreasantarlasci/Apps/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 1590, in dispatch_request
    self.raise_routing_exception(req)
  File "/Users/andreasantarlasci/Apps/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 1573, in raise_routing_exception
    raise request.routing_exception
  File "/Users/andreasantarlasci/Apps/microblog/venv/lib/python3.6/site-packages/flask/ctx.py", line 294, in match_request
    self.url_adapter.match(return_rule=True)
  File "/Users/andreasantarlasci/Apps/microblog/venv/lib/python3.6/site-packages/werkzeug/routing.py", line 1581, in match
    raise NotFound()
werkzeug.exceptions.NotFound: 404 Not Found: The requested URL was not found on the server.  If you entered the URL manually please check your spelling and try again.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/andreasantarlasci/Apps/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 1982, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/andreasantarlasci/Apps/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 1614, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/andreasantarlasci/Apps/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 1512, in handle_user_exception
    return self.handle_http_exception(e)
  File "/Users/andreasantarlasci/Apps/microblog/venv/lib/python3.6/site-packages/flask/app.py", line 1471, in handle_http_exception
    return handler(e)
  File "/Users/andreasantarlasci/Apps/microblog/app/errors.py", line 6, in not_found_error
    return render_template('404.html'), 404
  File "/Users/andreasantarlasci/Apps/microblog/venv/lib/python3.6/site-packages/flask/templating.py", line 134, in render_template
    context, ctx.app)
  File "/Users/andreasantarlasci/Apps/microblog/venv/lib/python3.6/site-packages/flask/templating.py", line 116, in _render
    rv = template.render(context)
  File "/Users/andreasantarlasci/Apps/microblog/venv/lib/python3.6/site-packages/jinja2/asyncsupport.py", line 76, in render
    return original_render(self, *args, **kwargs)
  File "/Users/andreasantarlasci/Apps/microblog/venv/lib/python3.6/site-packages/jinja2/environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "/Users/andreasantarlasci/Apps/microblog/venv/lib/python3.6/site-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/andreasantarlasci/Apps/microblog/venv/lib/python3.6/site-packages/jinja2/_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "/Users/andreasantarlasci/Apps/microblog/app/templates/base.html", line 4, in template
    {% if title %}{{ title }} - Microblog{% else %}{{ _(Welcome to Microblog!) }}{% endif %}
  File "/Users/andreasantarlasci/Apps/microblog/venv/lib/python3.6/site-packages/jinja2/environment.py", line 1005, in render
    return concat(self.root_render_func(self.new_context(vars)))
  File "/Users/andreasantarlasci/Apps/microblog/app/templates/404.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "/Users/andreasantarlasci/Apps/microblog/venv/lib/python3.6/site-packages/jinja2/environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/andreasantarlasci/Apps/microblog/venv/lib/python3.6/site-packages/jinja2/_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "/Users/andreasantarlasci/Apps/microblog/app/templates/base.html", line 4, in template
    {% if title %}{{ title }} - Microblog{% else %}{{ _(Welcome to Microblog!) }}{% endif %}
  File "/Users/andreasantarlasci/Apps/microblog/venv/lib/python3.6/site-packages/jinja2/environment.py", line 497, in _parse
    return Parser(self, source, name, encode_filename(filename)).parse()
  File "/Users/andreasantarlasci/Apps/microblog/venv/lib/python3.6/site-packages/jinja2/parser.py", line 901, in parse
    result = nodes.Template(self.subparse(), lineno=1)
  File "/Users/andreasantarlasci/Apps/microblog/venv/lib/python3.6/site-packages/jinja2/parser.py", line 883, in subparse
    rv = self.parse_statement()
  File "/Users/andreasantarlasci/Apps/microblog/venv/lib/python3.6/site-packages/jinja2/parser.py", line 130, in parse_statement
    return getattr(self, 'parse_' + self.stream.current.value)()
  File "/Users/andreasantarlasci/Apps/microblog/venv/lib/python3.6/site-packages/jinja2/parser.py", line 268, in parse_block
    node.body = self.parse_statements(('name:endblock',), drop_needle=True)
  File "/Users/andreasantarlasci/Apps/microblog/venv/lib/python3.6/site-packages/jinja2/parser.py", line 165, in parse_statements
    result = self.subparse(end_tokens)
  File "/Users/andreasantarlasci/Apps/microblog/venv/lib/python3.6/site-packages/jinja2/parser.py", line 883, in subparse
    rv = self.parse_statement()
  File "/Users/andreasantarlasci/Apps/microblog/venv/lib/python3.6/site-packages/jinja2/parser.py", line 130, in parse_statement
    return getattr(self, 'parse_' + self.stream.current.value)()
  File "/Users/andreasantarlasci/Apps/microblog/venv/lib/python3.6/site-packages/jinja2/parser.py", line 223, in parse_if
    drop_needle=True)
  File "/Users/andreasantarlasci/Apps/microblog/venv/lib/python3.6/site-packages/jinja2/parser.py", line 165, in parse_statements
    result = self.subparse(end_tokens)
  File "/Users/andreasantarlasci/Apps/microblog/venv/lib/python3.6/site-packages/jinja2/parser.py", line 875, in subparse
    add_data(self.parse_tuple(with_condexpr=True))
  File "/Users/andreasantarlasci/Apps/microblog/venv/lib/python3.6/site-packages/jinja2/parser.py", line 620, in parse_tuple
    args.append(parse())
  File "/Users/andreasantarlasci/Apps/microblog/venv/lib/python3.6/site-packages/jinja2/parser.py", line 432, in parse_expression
    return self.parse_condexpr()
  File "/Users/andreasantarlasci/Apps/microblog/venv/lib/python3.6/site-packages/jinja2/parser.py", line 437, in parse_condexpr
    expr1 = self.parse_or()
  File "/Users/andreasantarlasci/Apps/microblog/venv/lib/python3.6/site-packages/jinja2/parser.py", line 450, in parse_or
    left = self.parse_and()
  File "/Users/andreasantarlasci/Apps/microblog/venv/lib/python3.6/site-packages/jinja2/parser.py", line 459, in parse_and
    left = self.parse_not()
  File "/Users/andreasantarlasci/Apps/microblog/venv/lib/python3.6/site-packages/jinja2/parser.py", line 470, in parse_not
    return self.parse_compare()
  File "/Users/andreasantarlasci/Apps/microblog/venv/lib/python3.6/site-packages/jinja2/parser.py", line 474, in parse_compare
    expr = self.parse_math1()
  File "/Users/andreasantarlasci/Apps/microblog/venv/lib/python3.6/site-packages/jinja2/parser.py", line 496, in parse_math1
    left = self.parse_concat()
  File "/Users/andreasantarlasci/Apps/microblog/venv/lib/python3.6/site-packages/jinja2/parser.py", line 507, in parse_concat
    args = [self.parse_math2()]
  File "/Users/andreasantarlasci/Apps/microblog/venv/lib/python3.6/site-packages/jinja2/parser.py", line 517, in parse_math2
    left = self.parse_pow()
  File "/Users/andreasantarlasci/Apps/microblog/venv/lib/python3.6/site-packages/jinja2/parser.py", line 528, in parse_pow
    left = self.parse_unary()
  File "/Users/andreasantarlasci/Apps/microblog/venv/lib/python3.6/site-packages/jinja2/parser.py", line 547, in parse_unary
    node = self.parse_postfix(node)
  File "/Users/andreasantarlasci/Apps/microblog/venv/lib/python3.6/site-packages/jinja2/parser.py", line 676, in parse_postfix
    node = self.parse_call(node)
  File "/Users/andreasantarlasci/Apps/microblog/venv/lib/python3.6/site-packages/jinja2/parser.py", line 767, in parse_call
    self.stream.expect('comma')
  File "/Users/andreasantarlasci/Apps/microblog/venv/lib/python3.6/site-packages/jinja2/lexer.py", line 384, in expect
    self.name, self.filename)
jinja2.exceptions.TemplateSyntaxError: expected token ',', got 'to'
